util/build/enclave/src/lib.rs

```diff
@@ -590,16 +612,24 @@ impl Builder {
         // [3] https://github.com/slimm609/checksec.sh
         // [4] man ld(1)
 
-        if Command::new(&self.linker)
+        let mut command = Command::new(&self.linker);
+
+        command
             .args(&[
                 "-o",
                 unsigned_enclave
                     .to_str()
                     .expect("Invalid UTF-8 in unsigned enclave path"),
             ])
             .args(&["-z", "relro", "-z", "now", "-z", "noexecstack"])
-            .args(&["--no-undefined", "-nostdlib"])
-            .arg(format!("-L{}", self.libdir.display()))
+            .args(&["--no-undefined", "-nostdlib"]);
+
+        // We're only using link paths here, actual linkage is hard-coded.
```

Comment:
Can you explain a little more about this - where is the linkage hard-coded?

Reply:
It's the linker command in `Builder::link_unsigned()`.

Reply:
Ah I see - thank you :)